Author: Gabriel Filion <gabster@lelutin.ca>
Date:   Sun Feb 24 03:55:48 2019 -0500
Subject: Generate man page for smokeinfo
Forwarded: https://github.com/oetiker/SmokePing/pull/164
Applied-Upstream: e42f72c26d64d7247afff538ef808d01a73ad335

    The documentation is already there, the man page is juste currently not
    being generated.

--- smokeping-2.7.3.orig/doc/Makefile.am
+++ smokeping-2.7.3/doc/Makefile.am
@@ -38,7 +38,7 @@ MODBASE = $(subst .pm,,$(subst ../lib/Sm
 	  $(subst .pm,,$(subst ../lib/Smokeping/matchers/,Smokeping_matchers_,$(PODMATCH))) \
 	  $(subst .pm,,$(subst ../lib/Smokeping/sorters/,Smokeping_sorters_,$(PODSORT)))
 
-PROGBASE = smokeping smokeping_cgi tSmoke
+PROGBASE = smokeping smokeping_cgi tSmoke smokeinfo
 
 DOCSCONFIGBASE = smokeping_config smokeping_examples
 
@@ -89,6 +89,9 @@ smokeping_cgi.1: ../bin/smokeping_cgi
 tSmoke.1: ../bin/tSmoke
 	$(AM_V_GEN)$(POD2MAN) --section 1 > $@
 
+smokeinfo.1: ../bin/smokeinfo
+	$(AM_V_GEN)$(POD2MAN) --section 1 > $@
+
 smokeping_config.pod: ../lib/Smokeping.pm
 	$(AM_V_GEN)$(MAKEPOD) > $@
 
